manifests/map.pp

```diff
@@ -76,9 +80,13 @@
   $mappath_real = pick($mappath, "/etc/auto.${name}")
 
   # functionality
-  $content = $file ? {
-    undef   => template('autofs/mountmap.erb'),
-    default => undef,
+  if $manage_content {
+    $content = $file ? {
+      undef   => template('autofs/mountmap.erb'),
+      default => undef,
+    }
+  } else {
+    $content = undef
   }
```

Comment:
What about this alternative:

```
  if $manage_content == true and $file == undef {
    $content = template('autofs/mountmap.erb')
  } else {
    $content = undef
  }
```

I think its easier to read.

Reply:
I agree. Will update it. :)